docs/docs/guides/evaluation/builtin_scorers.mdx

```diff
@@ -15,7 +15,9 @@ import TabItem from '@theme/TabItem';
 
     **LLM-evaluators**
 
-    The pre-defined scorers that use LLMs support the OpenAI, Anthropic, Google GenerativeAI and MistralAI clients. They also use `weave`'s `InstructorLLMScorer` class, so you'll need to install the [`instructor`](https://github.com/instructor-ai/instructor) Python package to be able to use them. You can get all necessary dependencies with `pip install "weave[scorers]"`
+    The pre-defined scorers that leverage LLMs now automatically integrate with litellm.
+    You no longer need to pass an LLM client; just set the `model_id`. 
```

Comment:
I would prefix this with something like `Update Feb 2025:`

A user who just brand new to the ecosystem might be surprised by this comment otherwise